benchmarks/polygon.py

```diff
@@ -5,90 +5,90 @@
 from sympy.geometry import (Line, Point,
                             Polygon)
 from random import randint
-listOfPolygons = [Polygon(Point(0+w,0+w),
-                          Point(3+w,0+w),
-                          Point(3+w,3+w),
-                          Point(0+w,3+w)) for w in range(10)]
+listOfPolygons = [Polygon(Point(0+w, 0+w),
+                          Point(3+w, 0+w),
+                          Point(3+w, 3+w),
+                          Point(0+w, 3+w)) for w in range(10)]
 cutListOfPolygons = [Polygon((-1, -1), (1, Rational(5, 2)), (2, 1), (3, Rational(5, 2)), (4, 2), (5, 3), (-1, 3)) for w in range(10)]
 cutLines = [Line((0, 0), (Rational(9, 2), 3)) for w in range(10)]
 
 class PolygonAttributes:
     def time_create(self):
-        "Creating Polygon"
+        """Creating Polygon"""
```

Comment:
I think the reason for single quote is so the docstrings don't bog down the output to std. I wouldn't change these.

Reply:
I'll revert it.